RiotSwiftUI/Modules/Settings/ChangePassword/ChangePasswordViewModel.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+typealias ChangePasswordViewModelType = StateStoreViewModel<ChangePasswordViewState,
+                                                                       Never,
+                                                                       ChangePasswordViewAction>
+class ChangePasswordViewModel: ChangePasswordViewModelType, ChangePasswordViewModelProtocol {
+
+    // MARK: - Properties
+
+    // MARK: Private
+
+    // MARK: Public
+
+    var callback: (@MainActor (ChangePasswordViewModelResult) -> Void)?
+
+    // MARK: - Setup
+
+    init(oldPassword: String = "",
+         newPassword1: String = "",
+         newPassword2: String = "",
+         signoutAllDevices: Bool = true) {
```

Comment:
Based on the [ticket](https://github.com/vector-im/element-meta/issues/135#issue-1115215413) for the FTUE screens, this should probably default to unchecked as-well.

Reply:
Ah i missed that, was defaulted to `true` because the api was designed like that, updated now.